src/SmartComponents/DriftPage/DriftTable/__tests__/__snapshots__/DriftTable.tests.js.snap

```diff
@@ -11385,15 +11385,7 @@ exports[`ConnectedDriftTable should render systems, baselines and historicalProf
                                             />
                                           </div>
                                         }
-                                        footerContent={
-                                          <Button
-                                            onClick={[Function]}
-                                            ouiaId="hsp-popover-compare"
-                                            variant="primary"
-                                          >
-                                            Compare
-                                          </Button>
-                                        }
+                                        footerContent={null}
```

Comment:
There should probably also be a test to verify that the button does show when there is `historicalData` in the state.